react/tic-tac-toe/src/app/screens/App/components/Results/components/CustomRow/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class CustomRow extends Component {
+  renderItem = item => {
+    return <div>{item}</div>;
+  };
+
+  renderField = (field) => {
+    return <div key={this.props.rowData.id} className={`row${this.props.index}`}>{`${this.props.rowData[field]}`}</div>
+  }
+  render() {
+    const { rowData } = this.props;
+    return (
+      <>
+        {Object.keys(rowData).map(this.renderField)}
+      </>
+    );
+  }
+}
+
+CustomRow.propTypes = {
+  rowData: PropTypes.object,
```

Comment:
Use `shape` instead. With `shape` you define the default structure of this object. Check [this](https://www.npmjs.com/package/prop-types) out for more info.